drivers/pds/lib/utils.go

```diff
@@ -1093,6 +1104,59 @@ func CreateTempNS(length int32) (string, error) {
 	return namespace, nil
 }
 
+func isReachbale(url string) (bool, error) {
```

Comment:
there is a function isDashboardReachable in tests/common.go 
can we make that generic and use the same?

Reply:
I have checked it and we cannot use that method.